app/admin_statistics.py

```diff
@@ -32,9 +32,16 @@ def get_repo_usage(self, repo):
         Get num_persons and notes for specific repository
         Return: dictionary contains repository name,
         number of persons and notes, e.g.
-        {'repo': haiti, 'num_persons': 10, 'num_notes': 5}
+        {'repo': haiti, 'num_persons': 10, 'num_notes': 5, ...etc}
         """
         counters = model.UsageCounter.get(repo)
         return {'repo': repo,
-                'num_persons': counters.person_counter if counters else 0,
-                'num_notes': counters.note_counter if counters else 0}
+                'num_persons': getattr(counters, 'person', 0),
+                'num_notes': getattr(counters, 'note', 0),
+                'is_note_author': getattr(counters, 'is_note_author', 0),
```

Comment:
Simplify here with for-loop as well.

Also, you should prefix these keys with "num_notes" or something, otherwise it's not clear these are number of notes or persons with the status.